### ISSUE TITLE
feat: update toast component 

### ISSUE THREAD

**Issue by imtuyethan** [janhq/jan#1639]
# Problem:
The current toast kinda blends in with everything else and hard to see:
<img width="1191" alt="Screenshot 2024-01-17 at 10 42 22 AM" src="https://github.com/janhq/jan/assets/89722390/ae697189-0636-4baa-bffb-502122b758f7">

# Updated component
https://www.figma.com/file/Js8MF76jMQ99fDwtnOT8pA/%E2%9A%AB%EF%B8%8F--Design-System%3A-Joi?type=design&node-id=551-8011&mode=design&t=aWOWX3L8tR0Urmzk-4
<img width="507" alt="Screenshot 2024-01-17 at 12 02 24 PM" src="https://github.com/janhq/jan/assets/89722390/9a37e7d4-500e-4ac8-9ae6-bc507c0ed864">

**Comment by copyhold** [comment]
hi @urmauur , are you on this ticket?

**Comment by urmauur** [comment]
> hi @urmauur , are you on this ticket?

Yes, i'm still working in progress on this ticket, can you take a look another ticket? 🙏🏻
